### PR TITLE
Chapter 14: Following Users

### PR DESCRIPTION
**Pull request by megatimranikram**

### CODE REVIEW

**Comment by megatimranikram** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -187,3 +187,80 @@ aside {
...
+}
+
+.stats {
+  overflow: auto;
+  margin-top: 0;
+  padding: 0;
+  a {
+    float: left;
+    padding: 0 10px;
+    border-left: 1px solid $gray-lighter;
```
Comment (saobv-2223) [inline app/assets/stylesheets/custom.scss]: each block should have an empty line among them

File: app/controllers/relationships_controller.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  def create
+    @user = User.find_by id: params[:followed_id]
+    if @user.present?
+      current_user.follow @user
+    else
+      @error = t "follow.fail"
+    end
+    respond_to do |format|
+      format.js
+    end
```
Comment (saobv-2223) [inline app/controllers/relationships_controller.rb]:
```suggestion
    respond_to :js
```

File: app/controllers/relationships_controller.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    relationship = Relationship.find_by id: params[:id]
+    if relationship.present?
+      @user = relationship.followed
+      current_user.unfollow @user
+    else
+      @error = t "follow.fail"
+    end
+      respond_to do |format|
+      format.js
+    end
```
Comment (saobv-2223) [inline app/controllers/relationships_controller.rb:31]:
```suggestion
    respond_to :js
```

File: app/controllers/users_controller.rb [hunk 4]
```diff
@@ -52,6 +52,20 @@ def destroy
     redirect_to users_path
   end
 
+  def following
+    @title = "Following"
+    @user = User.find_by(id: params[:id])
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb]: remove ()

File: app/controllers/users_controller.rb [hunk 5]
```diff
@@ -52,6 +52,20 @@ def destroy
...
+    @users = @user.following.paginate(page: params[:page]).per Settings.per_page
+    render "show_follow"
+  end
+
+  def followers
+    @title = "Followers"
+    @user = User.find_by(id: params[:id])
```
Comment (saobv-2223) [inline app/controllers/users_controller.rb]: remove ()

**Comment by megatimranikram** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**